Content/scripts/weather_animation.lua

```diff
@@ -0,0 +1,45 @@
+runProcess(function()
```

Comment:
Sample scripts should follow how the other scripts work (comments, posting start and end messages on backlog)

Reply:
For the documentation of the binding functions, where should I put this?